tools/cldr-apps/js/src/esm/cldrLocales.mjs

```diff
@@ -4,8 +4,25 @@
 import * as cldrEvent from "./cldrEvent.mjs";
 import * as cldrLoad from "./cldrLoad.mjs";
 import * as cldrMenu from "./cldrMenu.mjs";
+import * as cldrNotify from "./cldrNotify.mjs";
 import * as cldrStatus from "./cldrStatus.mjs";
 import * as cldrSurvey from "./cldrSurvey.mjs";
+import * as cldrText from "./cldrText.mjs";
+
+// The current locale may be "USER" temporarily, meaning the back end should choose an appropriate locale
+// for the current user. The real locale ID should be set when a server response contains it.
+const USER_LOCALE_ID = "USER";
+
+const VALIDATE_ALL_LOCALES = true;
+const VALIDATE_ALL_LOCALES_VERBOSE = false;
+
+const LOCALE_REGEX = new RegExp("^[a-zA-Z0-9_]*$");
+
+let didValidateAll = false;
+
+// For unit testing, inhibit calling cldrNotify for locale-validation failures,
+// to avoid "global leak(s) detected: '__VUE__', '__VUE_DEVTOOLS_HOOK_REPLAY__'"
```

Comment:
instead, change mocha.setup in Test.html to include any symbols that are global. 

probably change `mocha.setup("bdd")` to `mocha.setup({ui:"bdd",global:['__VUE__','__VUE_DEVTOOLS_HOOK_REPLAY__']);`  see https://mochajs.org/#browser-configuration

i have hit this before, it's either on one of the PRs awaiting ticket acceptance or I ended up not seeing the leak.